rasa/core/channels/slack.py

```diff
@@ -348,19 +378,32 @@ def blueprint(
 
                 elif self._is_user_message(output):
                     metadata = self.get_metadata(request)
-                    return await self.process_message(
-                        request,
-                        on_new_message,
-                        self._sanitize_user_message(
-                            output["event"]["text"], output["authed_users"]
-                        ),
-                        output.get("event").get("user"),
-                        metadata,
-                    )
+                    if (
```

Comment:
what do we need this condition for?

Reply:
This is a holdover from the original codebase. I added the checks for direct message and app mention events. I think the response text should probably be set to "Received message on unsupported channel" or something to that effect instead of a blank response.